src/Helper/Placeholder.php

```diff
@@ -37,15 +36,12 @@ class Placeholder extends AbstractHelper
      * Placeholder helper
      *
      * @param  string $name
-     * @throws InvalidArgumentException
-     * @return Placeholder\Container\AbstractContainer
+     * @return Placeholder\Container\AbstractContainer|Placeholder
      */
     public function __invoke($name = null)
     {
         if ($name === null) {
-            throw new InvalidArgumentException(
```

Comment:
This particular change is most likely a BC break. Although I don't think people actually do rely on try-catching it, it is possible that they rely on the page to be crashing

Reply:
I totally agree with you. Should we move this to the `3.0.0` milestone?

Reply:
@Ocramius ping

Reply:
@thexpand 
Next major version makes sense, because nobody uses a try-catch-block here.